media/css/firefox/windows-64-bit.scss

```diff
@@ -13,21 +13,28 @@ $image-path: '/media/protocol/img';
     background: #33124e;
     color: #fff;
     text-align: center;
-    height: 400px;
+    height: 500px;
```

Comment:
Would this be better using `min-height` instead? The page is only in English for now, but should it be translated then a set height could potentially be too short for longer strings.